### DISCUSSION TITLE
how do i use frames

### DISCUSSION THREAD

**Discussion by RODIOMG7** [LibreSprite/LibreSprite#398] [Q&A]
how do i add thing to a frame????

**Comment by vladpereverzyev** [comment]
If you mean animation frames in LibreSprite, each frame represents a different image/state of the same layer.

To add something to a frame, select the frame in the timeline and then draw/paste your content normally on the corresponding layer. You can create a new frame using the options in the timeline (or duplicate an existing frame if you want to start from its contents).

If you want the same object to appear in multiple frames, you'll need to copy it to each frame, unless you're using a feature such as linked/continuous cels.

If you can clarify whether you mean adding a new frame, adding an image/object to an existing frame, or moving something between frames, I can give you the exact steps.

**Reply by RODIOMG7** [reply]
thanks for the help but you're a bit too early
